test/functional/cmdLineTests/criu/criu_onlyCheckpoint.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+
+<!--
+  Copyright (c) 2022, 2022 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
+
+<suite id="J9 Criu Command-Line Option Tests" timeout="300">
+  <variable name="MAINCLASS_SIMPLE" value="org.openj9.criu.CRIUSimpleTest" />
+
+  <test id="Create Criu Checkpoint Image">
+    <command>bash $SCRIPPATH$ $TEST_RESROOT$ $JAVA_COMMAND$ $JVM_OPTIONS$ $MAINCLASS_SIMPLE$ SingleCheckpoint OnlyCheckpoint</command>
```

Comment:
- This looks like it is duplicating https://github.com/eclipse-openj9/openj9/blob/18b0f7891874f134d83b3294b679804b3388a365/test/functional/cmdLineTests/criu/criu.xml#L31 with `OnlyCheckpoint`. Is it possible to add it to criu.xml?
- I would prefer to have a boolean for $6, not the string `OnlyCheckpoint`. True for keeping the test output and false for deleting `testOutput`.

Reply:
- `$6` has been updated to a boolean value as suggested.
- This test differs in output condition check from the referred one, e.g. "Post-checkpoint" output is considered as `failure` here since it should not restore checkpoint; while it is `success` for normal criu test. Also, if we merge these two xmls, since `OnlyCheckpoint` test and portable criu test only requires this single subtest, would we have ways to only run this subtest in playlist? Thanks.